apidoc/Titanium/UI/Label.yml

```diff
@@ -166,8 +166,13 @@ properties:
     summary: Minimum font size when the font is sized based on the contents. 
     description: |
         Enables font scaling to fit and forces the label content to be limited to a single line.
+        
+        On Android, setting this property will force the `ellipsize` property to be
+        <Titanium.UI.TEXT_ELLIPSIZE_TRUNCATE_END> which is the default behavior as well.
     type: Number
-    platforms: [iphone, ipad]
+    since: {iphone: "0.8.0", ipad: "0.8.0", android: "6.1.0"}
```

Comment:
Nope, it's random. Usually the API added first comes first and the new one is added to it. But there're really no guidelines for that one.